inst/json/metadata_0.1.0.json

```diff
@@ -60,24 +60,24 @@
       { "id": "education", "label": "Education" },
       { "id": "education_closures", "label": "Closures" },
       { "id": "education_absences", "label": "Absences" },
-      { "id": "life_years", "label": "Life Years" },
+      { "id": "life_years", "label": "Life years" },
       { "id": "life_years_infants", "label": "Infants" },
       { "id": "life_years_adolescents", "label": "Adolescents" },
-      { "id": "life_years_working_age", "label": "Working-age Adults" },
-      { "id": "life_years_retirement_age", "label": "Retirement-age Adults" }
+      { "id": "life_years_working_age", "label": "Working-age adults" },
+      { "id": "life_years_retirement_age", "label": "Retirement-age adults" }
     ],
     "capacities": [
-      { "id": "hospital_capacity", "label": "Hospital Capacity", "description": "Total hospital beds" },
-      { "id": "icu_capacity", "label": "ICU Capacity", "description": "ICU hospital beds" }
+      { "id": "hospital_capacity", "label": "Hospital capacity", "description": "Total hospital beds" },
+      { "id": "icu_capacity", "label": "ICU capacity", "description": "ICU hospital beds" }
     ],
     "interventions": [
       { "id": "school_closures", "label": "School closures" },
       { "id": "business_closures", "label": "Business closures" }
     ],
     "time_series": [
-      { "id": "cases", "label": "Cases", "description": "Number of confirmed cases" },
-      { "id": "vacc", "label": "Vaccinated", "description": "Cumulative vaccinations" },
-      { "id": "hosp_occ", "label": "Hospital Occupancy", "description": "Number of hospitalised cases" }
+      { "id": "infect", "label": "Infections", "description": "Infections not requiring hospitalisation" },
```

Comment:
Wondering about the possibility of a more accurate title for Infections too — best I have is “community infections”, what do you think?